Sources/LiCore/QueryUtil/LiQueryBuilder.swift

```diff
@@ -105,6 +106,12 @@ public class LiQueryBuilder {
             query.append(space)
             query.append(limitSetting)
         }
+        if let offsetSetting = liQuerySetting.offset {
+            query.append(space)
+            query.append(offset)
+            query.append(space)
+            query.append(offsetSetting)
```

Comment:
Isn't is possible to just format it using " \(offset) \(offsetSetting)" ?

Reply:
Yes, this is to keep it a bit clearer.

Reply:
I personally think appending it like this is a little more difficult to read and also more prone to crashes but I'll leave it upto you.